cmake/BLTPrivateMacros.cmake

```diff
@@ -245,6 +245,121 @@ macro(blt_setup_cuda_target)
     endif()
 endmacro(blt_setup_cuda_target)
 
+##------------------------------------------------------------------------------
+## blt_add_hip_library(NAME         <libname>
+##                     SOURCES      [source1 [source2 ...]]
+##                     HEADERS      [header1 [header2 ...]]
+##                     DEPENDS_ON   [dep1 ...]
+##                     LIBRARY_TYPE <STATIC, SHARED, OBJECT, or blank for executables>
```

Comment:
> blank for executables

Is this right? For ``blt_add_library``, omitting the library type reverts to the default/global build type.

Wouldn't one want to call ``blt_add_hip_executable``  for executables?

Reply:
Good catch. I've removed that from this comment.